AirTote/Services/ImagerySourceReader.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace AirTote.Services;
+
+public class ImagerySourceReader
+{
+	public string GroupName { get; set; }
+	public string ShortTitle { get; set; }
+	public string FullTitle { get; set; }
+	public string URL { get; set; }
+	public bool IsPublic { get; set; }
+
+	public async Task<string> ReadTextFileAsync(string filePath)
```

Comment:
インスタンスメンバーにアクセスしていないため、`static`にしてください。

```suggestion
	public static async Task<string> ReadTextFileAsync(string filePath)
```